site/data/roadmap.yaml

```diff
@@ -1,55 +1,55 @@
 - item: Networking overhaul
-  status: Finished
+  status: 🎉 Finished
   url: https://www.zaproxy.org/blog/2022-02-10-new-zap-networking-layer/
   year: 2022
 - item: Continue working on issues
-  status: In progress
+  status: 🚧 In progress
```

Comment:
Maybe 👷 instead or 🕑

This one kinda makes it seem like a road block.....

Reply:
I think the clock is too similar to the calendar and didnt want a specific character for inclusiveness...

Reply:
⚒️⌨️🛣🪜?

Reply:
🎬⚡

Reply:
⚒️ or ⚡ get my vote.